provider/awsprovider/awsprovider.go

```diff
@@ -36,6 +36,12 @@ func (p *AWSProvider) ConfigureRun(state *run.State) error {
 	}
 
 	state.RootURL = userData.RootURL
+	state.WorkerLocation = map[string]string{
+		"cloud":        "aws",
+		"instanceId":   iid_json.InstanceId,
+		"instanceType": iid_json.InstanceType,
+		"region":       iid_json.Region,
```

Comment:
This could be more specific and include the availability zone as well -- that might be useful to someone.